server/dist/routes/projects.js

```diff
@@ -4,22 +4,38 @@ Object.defineProperty(exports, "__esModule", {
   value: true
 });
 
-var _dbConnection = require("../services/dbConnection.js");
+exports.default = function (app) {
+  app.get("/projects", function (req, res) {
+    res.send("all projects");
+  });
 
-var _dbConnection2 = _interopRequireDefault(_dbConnection);
+  app.post("/projects", function (req, res) {
+    var _req$body = req.body,
```

Comment:
There's no need to put `req.body` in a variable here, especially that the variable signature is longer. On a side note, JS has a camel case convention.